markovs_household/input/appliances_input.py

```diff
@@ -0,0 +1,32 @@
+from abc import ABC, abstractmethod
+
+from markovs_household.data.appliance import ApplianceCategory, ApplianceType
+from markovs_household.data.household_categories import HouseholdType, HouseholdIncome
+
+
+class Input(ABC):
```

Comment:
Maybe we can find a little more descriptive name? `HouseholdApplianceStats` or something like this?